embassy-stm32/src/crs.rs

```diff
@@ -0,0 +1,67 @@
+use embassy_hal_common::PeripheralRef;
+
+use crate::pac::CRS as PAC_CRS;
+use crate::peripherals::CRS;
+use crate::rcc::enable_hsi48;
+use crate::rcc::sealed::RccPeripheral;
+use crate::{pac, Peripheral};
+
+pub struct Config {
+    sync_source: SyncSource,
```

Comment:
What other config options need to be added here? The default reset behavior is enough for USB. Do we need to support the other configuration stuff?